### ISSUE TITLE
Is this library compatible with "Glimmer" (Ember 1.13)?

### ISSUE THREAD

**Issue by brandonparsons** [huafu/ember-google-map#67]
Hi there,

I'm having some issues upgrading to 1.13, and wondering whether or not this component supports Glimmer. Hoping to narrow down my problems!

Thx :+1:

**Comment by caspg** [comment]
I think there is some issue with ember 1.13. After upgrading to ember 1.13.2 map displays fine, but markers don't show up (despite this msg in console`DEBUG: [google-maps] creating new marker: {title: Home, position: (14.766127, 102.81098700000007)}` ).

**Comment by asennikov** [comment]
Yeah, same in my app. There were discussion started for 2.0 support, but it's closed now - https://github.com/huafu/ember-google-map/issues/14

**Comment by lukemcfarlane** [comment]
+1 for adding 1.13 support now that it has been released.

**Comment by huafu** [comment]
I am not promising any date, but I will try to find some time very soon to update quickly my addon so that it can support 1.13, waiting for a better refactor when I'll have more free time.

**Comment by huafu** [comment]
Fixing this will actually also target #43

**Comment by huafu** [comment]
If anyone can help figuring out the last issues in #69 this could be closed very soon ;-)

**Comment by brandonparsons** [comment]
@huafu You probably have others waiting on this, but don't fix on my account. It turns out I only needed an extremely simple `google-map` component so I just rolled one myself.  I'll leave this issue open though, feel free to close if you like.

**Comment by huafu** [comment]
Haha yeah well I believe many others are waiting on this anyway.

**Comment by ConorNugent** [comment]
I just needed a really simple subset of google maps component so I ended up rolling my own component. I am running into issues with ember 1.13.5 and markers not displaying though. I noticed that a bunch of people on this thread have experienced the same issue. Does anyone know what's behind it and whether there is a fix? Is it glimmer related?

Sorry, if this isn't the most ideal place to post about this issue but it was the only place came across people experiencing the same problem

**Comment by huafu** [comment]
I fixed them already in https://github.com/huafu/ember-google-map/issues/69 but aparently there are some more issues I need to fix there related to the `beforeObservers` that have been removed in `1.13`

**Comment by ConorNugent** [comment]
Ah okay, do you know what is causing the issue with markers not appearing in 1.13? I'm just trying to fix the same issue in my own very basic mapping component

**Comment by huafu** [comment]
Well if you did your own component, then I believe it can't be done the same way as mine, so I don't see how I could help on this ;-)

**Comment by kkincade** [comment]
Are Google options working with 1.13.5? I'm struggling with such options as gopt_zoomControl=false.

**Comment by ConorNugent** [comment]
@huafu you are dead right and I got things sorted. My issue was all down to something silly. Initially I thought it might have been a 1.13 issue hence the message here. Anyway, I'm really sorry for having added noise to your issues

**Comment by oswaldoacauan** [comment]
`gopt_disableDefaultUI=true` is not working with Ember 1.13.3.

cc @kkincade @huafu

**Comment by huafu** [comment]
Can you confirm the options issue with latest 0.0.24?

**Comment by kkincade** [comment]
@huafu After updating to 0.0.24, I'm still having this issue.
